applications/dashboard/controllers/class.settingscontroller.php

```diff
@@ -8,6 +8,7 @@
  * @since 2.0
  */
 use Vanilla\Addon;
+use Vanilla\Web\HttpStrictTransportSecurity\HttpStrictTransportSecurityModel as HstsModel;
```

Comment:
What was your reasoning for not just calling this `HstsModel`? Bit of mouthful eh?

Reply:
I prefer to have descriptive name to understand what is the thing. Not sure what is our standards and policy to use abbreviation for class names. 
To me there are not that many places where I need to call this class.
I've set up local short alias for this just for settingscontroller, because it was difficult to read the code with long name. In all other cases that was ok to keep full name for this class.

Reply:
Sounds good.